nightwatch/conf/nightwatch.conf.js

```diff
@@ -0,0 +1,67 @@
+const config = {
+  "nightwatchConfig": {
+      "page_objects_path": "nightwatch/pageObjects",
+
+      "selenium": {
+        "start_process": false,
+        "host": "hub-cloud.browserstack.com",
+        "port": 80
+      },
+
+      "common_capabilities": {
+        'browserstack.user': process.env.BROWSERSTACK_USERNAME,
+        'browserstack.key': process.env.BROWSERSTACK_ACCESS_KEY,
+        'browserstack.debug': true,
+        'browserstack.local': true,
+        'resolution': '1920x1080'
+      },
+
+      "test_settings": {
+        "default": {},
+        "chrome": {
+          "desiredCapabilities": {
+            "browser": "chrome",
```

Comment:
We need to nail down min requirements here - also do you need to specify which browsers to test against in the run command if they are specified in this config file?